api/scripts/daily-digest.ts

```diff
@@ -38,14 +31,14 @@ import { Op } from "sequelize";
         'id': group.GroupId
```

Comment:
Looks like you're making the incorrect assumption that there's only one group/project per user.  I think we'd want to be breaking down the digest for *all* of the users projects individually, but in a single email.